src/components/AppHeader.js

```diff
@@ -5,14 +5,24 @@ import * as Actions from '../actions/mapActions'
 import { convertBranch } from '../actions/appActions'
 
 const columnLinks = props => {
-  const { id, name, click, active } = props
-  const style = () => (active) ? 'is-active' : ''
+  const { id, name, click } = props
+  const style = () => {
+    const base = 'nav-item '
+    switch (id) {
+      case 0:
+        return base + 'gm-navA'
+      case 1:
+        return base + 'gm-navB'
+      case 2:
+        return base + 'gm-navC'
+      default:
+        return base
+    }
+  }
```

Comment:
👍 Looks good. My ruby brain wants to complain about having this wrapped in a function, but this is actually a clever way to do it in JS.